projects/observability/src/shared/components/top-n/top-n-chart.component.ts

```diff
@@ -1,18 +1,31 @@
 import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnChanges, Output } from '@angular/core';
 import { TypedSimpleChanges } from '@hypertrace/common';
-import { sortBy } from 'lodash';
-import { HistogramBarData } from '../histogram/histogram-chart.component';
+import { maxBy } from 'lodash';
 
 @Component({
   selector: 'ht-top-n-chart',
+  styleUrls: ['./top-n-chart.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <ht-histogram-chart
```

Comment:
Do we want the histogram and top-n diverging, or should this change be to the underlying histogram component?

Reply:
Yes. IMO it should diverge since now top-n is significantly different from the histogram/bar chart. Histogram may still have usage is some cases. So let's roll with this and probably we can come back to this later after few months.